xchange-jubi/src/main/java/org/knowm/xchange/jubi/service/JubiMarketDataService.java

```diff
@@ -25,16 +26,18 @@ public JubiMarketDataService(Exchange exchange) {
   }
 
   @Override
-  public Ticker getTicker(CurrencyPair currencyPair,
-      Object... args) throws ExchangeException, NotAvailableFromExchangeException, NotYetImplementedForExchangeException, IOException {
-
+  public Ticker getTicker(CurrencyPair currencyPair, Object... args)
+          throws ExchangeException, NotAvailableFromExchangeException, NotYetImplementedForExchangeException, IOException {
     // Request data
     JubiTicker jubiTicker = getJubiTicker(currencyPair.base.getCurrencyCode(), currencyPair.counter.getCurrencyCode());
-
     // Adapt to XChange DTOs
     return jubiTicker != null ? JubiAdapters.adaptTicker(jubiTicker, currencyPair) : null;
   }
 
+  public Map<String, Ticker> getAllTicker(Object... args) throws IOException {
```

Comment:
Let's keep this out of the generic service for now. To access this, just cast to the raw service. in the future, if we add a method like this to the generic service, the map should probably be `<CurrencyPair, Ticker>`.